app/src/main/java/com/zendesk/rememberthedate/storage/AppStorage.java

```diff
@@ -32,7 +35,7 @@
 
     public AppStorage(Context context) {
         this.storage = context.getSharedPreferences(REMEMBER_THE_DATE_STORE, Context.MODE_PRIVATE);
-        this.gson = new Gson();
+        this.gson = new GsonBuilder().setDateFormat(Constants.ISO8601_DATE_PATTERN).create();
```

Comment:
Can you check if this is still needed, @KillianSmith0? I would assume that since we're not storing a `Date` anymore, we shouldn't need to define a format for serialising dates in GSON. It'd be nice to not have to do this bit, in the mindset of "the best code is the code you don't have to write".

Reply:
Ah yes, okay that makes sense!